website/sidebars.json

```diff
@@ -9,25 +9,25 @@
     "MobX core": [
       "refguide/observable",
       "refguide/action",
-      "refguide/autorun",
-      "refguide/computed"
+      "refguide/computed",
+      "refguide/autorun"
     ],
     "MobX and React": [
-      "react/react-integration"
+      "react/react-integration",
+      "react/react-performance"
     ],
     "Tips & Tricks": [
+      "best/store",
       "best/what-does-mobx-react-to",
       "best/debugging-mobx",
-      "best/store",
-      "react/react-performance",
-      "refguide/mobx-utils",
       "refguide/computed-with-args",
+      "refguide/mobx-utils",
```

Comment:
I've moved it to after Computed with args, since there are 3 other ways to do things before resorting to mobx-utils, as per computed-withargs.